build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    java
+    kotlin("jvm") version "1.4.21"
```

Comment:
Why Kotlin the lang is amazing yes but it's unused here LOL
Also Kotlin 1.4.21?? we're at 1.4.31 now